workers/data_refinery_workers/processors/smasher.py

```diff
@@ -117,20 +119,43 @@ def _prepare_files(job_context: Dict) -> Dict:
     log_state("end prepare files", job_context["job"], start_prepare_files)
     return job_context
 
+def downlad_computed_file(item):
+    """ this function downloads the latest computed file sent in it's arguments,
+    its used to parallelize the download of all files. """
+    (output_file_path, latest_computed_file) = item
```

Comment:
Could you improve the docstring and/or parameter name here? The docstring isn't grammatical and it's a tad vague and combined with a parameter named `item` it's rather vague. Perhaps you could typehint the Tuple parameter: https://docs.python.org/3/library/typing.html#typing.Tuple?

Reply:
Btw, you're using `it's` and its` backwards:
  `it's` == `it is`
  `its` is possessive